### DISCUSSION TITLE
blog/2022/06/login-with-apple-workaround/

### DISCUSSION THREAD

**Discussion by giscus** [d0lim/blog-comments#9] [Announcements]
# blog/2022/06/login-with-apple-workaround/

애플 로그인을 구현하기 위해 필요한 Backend 시퀀스와 로직

https://d0lim.com/blog/2022/06/login-with-apple-workaround/

**Comment by gatherKnowledge** [comment]
머리에 깔끔하게 들어오내요. 글 잘봤습니다

**Reply by d0lim** [reply]
감사합니다!
